src/common/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface IIssue {
+  id: number;
+  name: string;
+  description: string;
+  status: string;
+  assignee: string;
+}
+
+export type UpdateStatusPaylod = {
```

Comment:
Use mainly interfaces for abstraction. If you can't then use type. A name for this file I have seen is models.ts or interfaces.ts